### PR TITLE
add icon for windows build

### PR DESCRIPTION
**Pull request by z3t0**
#334

not tested

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -128,6 +128,7 @@ module.exports = function (grunt) {
 					'app-version': version,
 					platform: 'win32',
 					arch: 'all',
+					icon:'icons/icon256.ico"
```
Comment (israelroldan) [inline Gruntfile.js]: you're mixing double quotes and single quotes

**Comment by z3t0** [comment]
@israelroldan fixed. I was editing it in browser

**Comment by PalmerAL** [comment]
I'm still seeing the default icon for the Windows build:

screen shot 2017-04-16

According to the electron-packager documentation, this should work, so I'm not sure why it's not.

**Comment by z3t0** [comment]
@PalmerAL yeah no idea, i run on mac so no way to test hmm.

**Comment by PalmerAL** [comment]
You can get a Windows VM from https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/

**Comment by PalmerAL** [comment]
Testing this again, I think this is happening because of the windows icon cache - if I move the exe to a different location, the icon shows up correctly.

**Comment by PalmerAL** [comment]
Merged in 90fcf8860dcf3f344944051bbaeb4bb6e91ed760 because I can't figure out how to resolve the merge conflicts from this branch.

**Pull request closed without merging**